README.md

```diff
@@ -75,42 +73,17 @@ This will mount the ./dist folder into the supervisor container and build the co
 ### Testing with preloaded apps
 To test preloaded apps, add a `tools/dind/apps.json` file according to the preloaded apps spec.
```

Comment:
The wording here needs to be a little different if we aren't showing an example of what the file should look like. "To test preloaded apps, run `resin preload`...and copy the `apps.json` file from the `resin-data` partition of the preloaded image to `tools/dind/`."

Reply:
Done